### PR TITLE
Preset keywords

### PR DESCRIPTION
**Pull request by Kingdorian**
I added preset keywords to the preset class.

### CODE REVIEW

**Comment by bvanwijk** [comment]
Looks good, and well tested.

File: backend/src/main/java/com/benine/backend/Preset.java [hunk 1]
```diff
@@ -188,23 +256,11 @@ public boolean equals(Object o) {
     if (autofocus != preset.autofocus) {
       return false;
     }
-    
+    if (!keyWords.equals(preset.getKeywords())) {
```
Comment (bvanwijk) [inline backend/src/main/java/com/benine/backend/Preset.java]: You updated the equals method but you didn't update the hashcode method

**Comment by jochem725** [comment]
Looks fine, maybe we should keep track of all keywords used, so we can provide this to the search interface later.

**Comment by NaomidR** [comment]
Good that you added a list with the keywords. We'll need this later. Looks good

**Comment by bvanwijk** [comment]
I think this can be merged.

**Pull request merged**